sdk/identity/Azure.Identity/src/IExtendedTokenCredential.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core;
+
+namespace Azure.Identity
+{
+    internal interface IExtendedTokenCredential
+    {
+        Task<(AccessToken, Exception)> GetTokenAsync(TokenRequestContext requestContext, CancellationToken cancellationToken);
```

Comment:
This should be ValueTask

Reply:
merged in https://github.com/Azure/azure-sdk-for-net/issues/8317

Reply:
Done